Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -579,6 +588,11 @@ public void addAllObjectsToIndex() throws CustomResponseException, InterruptedEx
         indexer.performMultipleRequests(findAll(), processType);
     }
 
+    @Override
+    public ProcessDTO convertJSONObjectToDTO(JSONObject jsonObject, boolean related) throws DataException {
+        return null;
```

Comment:
Please
- do not return a `null` value, use an empty expected object like on other places.
- add a TODO annotation that mentioned that this method is not full implemented.